pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
```

Comment:
```suggestion
```

This doesn't belong here; `AsyncAPI Style Guide` will be a section title on the left-hand navigation. A great example of how this will look is the Generator Tool docs section:

<img width="1225" alt="Screen Shot 2023-02-21 at 5 52 06 PM" src="https://user-images.githubusercontent.com/19964402/220500705-a4760ead-f6c2-49d4-95fb-e69a1db0929f.png">